CMakeLists.txt

```diff
@@ -234,6 +237,21 @@ if(LIBDEFLATE_BUILD_STATIC_LIB)
     set_target_properties(libdeflate_static PROPERTIES
                           OUTPUT_NAME ${STATIC_LIB_NAME}
                           PUBLIC_HEADER libdeflate.h)
+    if(LIBDEFLATE_FRAMEWORK)
+        set_target_properties(libdeflate_static PROPERTIES
```

Comment:
Is it correct for this to be specific to the static library, not the shared one?

Reply:
Dynamic libs aren't allowed on iOS.